### PR TITLE
Fix lab link

### PR DESCRIPTION
**Pull request by squirrelo**
Tiny PR to change lab link from Colorado to UCSD

### CODE REVIEW

File: amgut/lib/locale_data/american_gut.py [hunk 1]
```diff
@@ -173,7 +173,7 @@
...
+    'NOT_A_BUSINESS_ANS': 'We have had many enquiries about our "service" or "business". %(shorthand)s is a contribution-supported academic project that is a collaboration between the <a href="http://www.earthmicrobiome.org">Earth Microbiome Project</a> and the <a href="http://humanfoodproject.com/">Human Food Project</a>, primarily run out of the <a href="https://knightlab.ucsd.edu/">Knight Lab</a> at the University of California San Diego, and is not a business or service.  In particular, %(shorthand)s is not a diagnostic test (although the information gained through the project may in future contribute to the development of diagnostic tests). All data except for information that needs to be kept confidential for privacy reasons is openly and freely released into public databases, and the project is not intended to make a profit (any surplus funds would be recycled back into furthering human microbiome research).' % {"shorthand": AMGUT_CONFIG.project_shorthand},
```
Comment (EmbrietteH) [inline amgut/lib/locale_data/american_gut.py]: "enquiries" --> "inquiries"

File: amgut/lib/locale_data/british_gut.py [hunk 2]
```diff
@@ -486,7 +486,7 @@
...
+    'NOT_A_BUSINESS_ANS': 'We have had many enquiries about our "service" or "business". %(project_shorthand)s is a contribution-supported academic project that is a collaboration between the <a href="http://www.earthmicrobiome.org">Earth Microbiome Project</a> and the <a href="http://humanfoodproject.com/">Human Food Project</a>, primarily run out of the <a href="https://knightlab.ucsd.edu/">Knight Lab</a> at the University of California San Diego, and is not a business or service.  In particular, the project is not a diagnostic test (although the information gained through the project may in future contribute to the development of diagnostic tests). All data except for information that needs to be kept confidential for privacy reasons is openly and freely released into public databases, and the project is not intended to make a profit (any surplus funds would be recycled back into furthering human microbiome research).' % {'project_shorthand': AMGUT_CONFIG.project_shorthand},
```
Comment (EmbrietteH) [inline amgut/lib/locale_data/british_gut.py]: "enquiries" --> "inquiries"

**Comment by EmbrietteH** [comment]
One minor comment; otherwise 👍

File: amgut/lib/locale_data/british_gut.py (hunk 2 above)
Comment (squirrelo) [inline amgut/lib/locale_data/british_gut.py]: British spelling for british gut.

**Pull request merged**